app/react/src/server/options.js

```diff
@@ -3,8 +3,8 @@ import packageJson from '../../package.json';
 export default {
   packageJson,
   frameworkPresets: [
-    require.resolve('./framework-preset-react.js'),
+    require.resolve('@storybook/addon-react/dist/server/framework-preset-react.js'),
```

Comment:
I think it's better to have the presets more accessible, something like: 

```js
import { presets } from '@storybook/addon-react';

/* blah blah blah */

frameworkPresets: [
  presets.react,
  require.resolve('./framework-preset-cra.js'),
  presets.reactDocgen,
]
```

also it will be more formatted way for frameworks to expose their presets.

Another option will be hide the particular presets, for example:

```js
import { createPresets } from '@storybook/addon-react';

/* blah blah blah */

frameworkPresets: [
  ...createPresets({
     afterReact: [require.resolve('./framework-preset-cra.js')],
  });
]
```

Reply:
I don't think we can mix preview and server code in one endpoint, so it probably can't be just `@storybook/addon-react`.

I think I'll make it just `@storybook/addon-react/preset` after we merge #6828